app/controllers/password_resets_controller.rb

```diff
@@ -0,0 +1,58 @@
+class PasswordResetsController < ApplicationController
+  before_action :get_user, only: [:edit, :update]
+  before_action :valid_user, only: [:edit, :update]
+  before_action :check_expiration, only: [:edit, :update]
```

Comment:
```suggestion
  before_action :get_user, :valid_user, :check_expiration, only: [:edit, :update]
```
ri được hì 
vì 3 method dùng chung cho 2 action nè 
ngoài ra, tên method ko dc bắt đầu bằng set/get em nhé, vì đôi lúc nó trùng với các method core của ruby và bị hiểu nhầm thành em đang override method á